cmd/validate/dependencies.go

```diff
@@ -32,10 +32,6 @@ func NewDependenciesCmd(furyctlBinVersion string) *cobra.Command {
 			if !ok {
```

Comment:
debug is mandatory?

Reply:
@Al-Pragliola

Reply:
seems like it's not. It was a parameter in distro Downloader, but there was not used